java-spiffe-core/src/main/java/spiffe/bundle/jwtbundle/JwtBundleSource.java

```diff
@@ -0,0 +1,21 @@
+package spiffe.bundle.jwtbundle;
+
+import lombok.NonNull;
+import spiffe.exception.BundleNotFoundException;
+import spiffe.spiffeid.TrustDomain;
+
+/**
+ * A <code>JwtBundleSource</code> represents a source of JWT bundles.
+ */
+public interface JwtBundleSource {
```

Comment:
I realized that JwtBundleSource and X509BundleSource are actually the same interface with the only difference being the return type of the getXxxBundleForTrustDomain.
Would it make sense to you to replace JwtBundleSource and X509BundleSource for the following?
```java
public interface BundleSource<T> {
    T getBundleForTrustDomain(@NonNull final TrustDomain trustDomain) throws BundleNotFoundException;
}
```

Reply:
Good one! I made that refactor.